### ISSUE TITLE
Error in the browser console

### ISSUE THREAD

**Issue by cedx** [sab-international/rich-text-editor#1]
When I try to use the editor, the browser console displays :

```
toolbarview-item-unavailable: The requested toolbar item is unavailable.
```

**Comment by cedx** [comment]
This is due to the toolbar items: the `fontSize` one is used but not included in the final bundle. It's a bug: the `Font` plugin is not enabled, so the `fontSize` toolbar item is not available.

**Comment by cedx** [comment]
Fixed in version 1.0.1.
